web/src/routes/index.tsx

```diff
@@ -0,0 +1,16 @@
+import {Card} from '@/components/ui/card';
+import {createFileRoute} from '@tanstack/react-router';
+
+export const Route = createFileRoute('/')({
+  component: RouteComponent,
+});
+
+/**
+ * RouteComponent component renders the home page.
+ * It displays a card with the home page content.
+ *
+ * @returns {JSX.Element} The rendered RouteComponent component.
+ */
+function RouteComponent() {
+  return <Card>Home</Card>;
```

Comment:
Home should have some content. I created a new home page for old Conductor with links to the web app etc.  Also need to have an option to copy a user token for scripts somewhere - currently on the old conductor home page - that could be in the user profile.